File: VB/E3470.Web/CustomFactoryService.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpf.Printing.Service.Native.Services
Imports DevExpress.XtraPrinting

Namespace E3470.Web
	Public Class CustomFactoryService
		Inherits FactoryService
		Public Sub New(ByVal container As IIntegrityContainer)
			MyBase.New(container)
		End Sub
		Protected Overrides Sub EnableXpsExport(ByVal printingSystem As PrintingSystemBase)
		End Sub
	End Class
End Namespace

File: CS/E3470.Web/CustomFactoryService.cs
using DevExpress.Xpf.Printing.Service.Native.Services;
using DevExpress.XtraPrinting;

namespace E3470.Web {
    public class CustomFactoryService : FactoryService {
        public CustomFactoryService(IIntegrityContainer container)
            : base(container) {
        }
        protected override void EnableXpsExport(PrintingSystemBase printingSystem) {
        }
    }
}